cmd/fleet/missing.txt

```diff
@@ -1,3 +1,5 @@
+# TODO: Can we convert this to Github issues, still needed?
```

Comment:
@ph Seems you created this file. I think we should convert these to gibhub issues. We have stats, what about the others?

Reply:
I think this come from the original directory when I've closed @scunningham's repository. 
I am not sure what "run as admin; doesn't drop creds" mean? is that dropping privileges?

Reply:
A bit cryptic on my part.  That was a note to myself that there is no reason for the fleet server to run with Elastic admin privileges.  We are addressing that with service accounts.

Reply:
I removed it.